### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by AmrTamer23**

### CODE REVIEW

**Review by Mahmoudgalalz — APPROVED** [review]

File: src/components/event/ticket-reqs/send-invitaion-modal.tsx [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+        email: data.email,
+        number: data.phone,
+        type:
+          data.type === "free"
+            ? data.ticketId === "Family and Friends"
+              ? "Family and Friends"
+              : "invitation - " + data.ticketId
+            : "paid",
+        eventId: data.eventId,
+        ...(data.type === "free" ? {} : { ticketId: data.ticketId }),
```
Comment (Mahmoudgalalz) [inline src/components/event/ticket-reqs/send-invitaion-modal.tsx:97]: We need to change Family to  Invitation Lounge,

And make sure to capitalize first I

File: src/components/event/ticket-reqs/send-invitaion-modal.tsx [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+                      <FormControl>
+                        <SelectTrigger>
+                          <SelectValue placeholder="Select a Type" />
+                        </SelectTrigger>
+                      </FormControl>
+                      <SelectContent>
+                        <SelectItem value="Regular">Regular</SelectItem>
+                        <SelectItem value="VIP">VIP</SelectItem>
+                        <SelectItem value="Family and Friends">
+                          Family and Friends
```
Comment (Mahmoudgalalz) [inline src/components/event/ticket-reqs/send-invitaion-modal.tsx:241]: Here

File: src/components/login-form.tsx [hunk 3]
```diff
@@ -33,6 +33,13 @@ export function LoginForm() {
     },
   });
 
+  if (typeof window !== "undefined") {
+    const token = localStorage.getItem("token");
+    if (token?.length > 10) {
+      window.location.href = "/dashboard";
+    }
+  }
+
```
Comment (Mahmoudgalalz) [inline src/components/login-form.tsx:42]: Very bad and has a lot of issues

**Review by Mahmoudgalalz — CHANGES_REQUESTED** [review]

File: src/components/login-form.tsx (hunk 3 above)
Comment (AmrTamer23) [reply src/components/login-form.tsx:42]: I know but to enhance this we need an endpoint to verify on the backend, can I work on it? Or do you have a better approach?

**Pull request merged**